packages/dht/test/simulation/data/generatedhtids.ts

```diff
@@ -0,0 +1,94 @@
+import fs from 'fs'
```

Comment:
Remove some commented-out code from this file

Reply:
removed